src/main/java/org/zalando/intellij/swagger/file/SwaggerUiCreator.java

```diff
@@ -61,34 +60,28 @@ private void setSwaggerConfigurationValues(
   private void copyFromJar(final Path target) throws URISyntaxException, IOException {
     String resourcePath = "/" + SWAGGER_UI_FOLDER_NAME;
     URI resource = Objects.requireNonNull(getClass().getResource(resourcePath), "resource").toURI();
-    FileSystem fileSystem =
-        FileSystems.newFileSystem(resource, Collections.<String, String>emptyMap());
-
-    final Path jarPath = fileSystem.getPath(resourcePath);
-
-    Files.walkFileTree(
-        jarPath,
-        new SimpleFileVisitor<>() {
-
-          @Override
-          public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs)
-              throws IOException {
-            final Path currentTarget = target.resolve(jarPath.relativize(dir).toString());
-            Files.createDirectories(currentTarget);
-            return FileVisitResult.CONTINUE;
-          }
-
-          @Override
-          public FileVisitResult visitFile(Path file, BasicFileAttributes attrs)
-              throws IOException {
-            Files.copy(
-                file,
-                target.resolve(jarPath.relativize(file).toString()),
-                StandardCopyOption.REPLACE_EXISTING);
-            return FileVisitResult.CONTINUE;
-          }
-        });
-
-    fileSystem.close();
+    try (FileSystem fileSystem = FileSystems.newFileSystem(resource, Map.of())) {
+
+      final Path jarPath = fileSystem.getPath(resourcePath);
+
+      try (Stream<Path> stream = Files.walk(jarPath)) {
```

Comment:
```suggestion
      try (Stream<Path> stream = Files.walk(fileSystem.getPath(resourcePath))) {
```

Reply:
You don't use the variable later, so why actually assign it at all?!

Reply:
> You don't use the variable later, so why actually assign it at all?!

Would be nice, but it's actually used (in lines 72 and 77).

Reply:
So much for my eyesight. :smirk: